test/spec/controllers/about.js

```diff
@@ -1,58 +1,54 @@
-'use strict';
+/* eslint-env mocha */
+'use strict'
 /* jshint expr: true */
 /* global sinon: false */
 
 describe('Controller: AboutCtrl', function () {
-
   // load the controller's module
-  beforeEach(module('openhimConsoleApp'));
+  beforeEach(module('openhimConsoleApp'))
 
   // setup config constant to be used for API server details
-  beforeEach(function() {
-    module('openhimConsoleApp', function($provide){
-      $provide.constant('config', { 'version': '1.7.0', 'minimumCoreVersion':'3.0.0', 'protocol': 'https', 'host': 'localhost', 'port': 8080, 'title': 'Title', 'footerTitle': 'FooterTitle', 'footerPoweredBy': 'FooterPoweredBy' });
-    });
-  });
-
-  var scope, createController, httpBackend, modalSpy;
-  
+  beforeEach(function () {
+    module('openhimConsoleApp', function ($provide) {
+      $provide.constant('config', { 'version': '1.7.0', 'minimumCoreVersion': '3.0.0', 'protocol': 'https', 'host': 'localhost', 'port': 8080, 'title': 'Title', 'footerTitle': 'FooterTitle', 'footerPoweredBy': 'FooterPoweredBy' })
+    })
+  })
+
+  var scope, createController, httpBackend, modalSpy // eslint-disable-line 
```

Comment:
@tumijacob Why is this line disabled?

Reply:
@bausmeier It was complaining about modalSpy not being used. It's used in another file.

Reply:
@tumijacob This is a local variable. It shouldn't be used in another file. If that is the case though and it can't be fixed properly then at the very least we should only disable that specific rule on this line.

Reply:
:+1: